kpatch-build/create-diff-object.c

```diff
@@ -226,13 +236,12 @@ static void kpatch_detect_child_functions(struct kpatch_elf *kelf)
 			if (!pname)
 				ERROR("strndup");
 
-			sym->parent = find_symbol_by_name(&kelf->symbols, pname);
+			parent = find_symbol_by_name(&kelf->symbols, pname);
 			free(pname);
 
-			if (!sym->parent)
-				ERROR("failed to find parent function for %s", sym->name);
+			if (!parent)
```

Comment:
Are we missing ERROR("failed to find parent function for %s", parent) here?

Reply:
Yes, seems I messed my patch splitting, thanks for spotting this!

I'll update as well.